src/api/batch/farmlands.md

```diff
@@ -0,0 +1,197 @@
+---
+layout: default
+title: farmlands-external-asset
+parent: Batches
+grand_parent: Api Reference
+has_children: false
+nav_exclude: true
+permalink: /api/batch-types/farmlands-external-asset
+description: |
+  Load Farmlands card data
+---
+
+# Farmlands Batch File
+This page details the expected file format for Farmlands [Batch Loading][].
+
+## Contents
+{:.no_toc .text-delta}
+
+* TOC
+{:toc}
+
+## Models
+A batch file will consist of a list of [Accounts][].
+
+<a name="account">
+### Account
+
+{% h4 Required Fields %}
+
+|     Field     |                   Description                    |
+| :------------ | :----------------------------------------------- |
+| externalId    | External unique identifier for Farmlands account |
+| accountNumber | Farmlands account number                         |
+
+
+{% h4 Optional Fields %}
+
+|      Field       |                                 Description                                  |
+| :--------------- | :--------------------------------------------------------------------------- |
+| availableBalance | Current spendable balance of account in cents                                |
+| contacts         | List of account [Contacts][]                                                 |
+
+<a name="contact">
+### Contact
+
+{% h4 Required Fields %}
+
+|   Field    |  Type  |              Description               |
+| :--------- | :----- | :------------------------------------- |
+| externalId | String | External unique identifier for contact |
+
+
+{% h4 Optional Fields %}
+
+|   Field   |  Type   |                                        Description                                        |
+| :-------- | :------ | ----------------------------------------------------------------------------------------- |
+| name      | String  | Name of contact                                                                           |
+| mobile    | String  | Mobile number of contact                                                                  |
+| email     | String  | Email address of contact                                                                  |
+| isPrimary | Boolean | 'true' if the contact is the primary contact of the containing account, 'false' otherwise |
```

Comment:
"isPrimary" should be "primary".  Use backticks for boolean literals. "false otherwise" is redundant.